### DISCUSSION TITLE
XCode console log full of "To Native -> CapacitorSQLite" output

### DISCUSSION THREAD

**Discussion by leemcdole** [capacitor-community/sqlite#517] [General]
I'm getting some undesired logging output in XCode whenever I make a query, I'm not sure if there's a way to disable it?  Or if I've somehow inadvertently enabled it?  
I'll get the following:

⚡️  To Native ->  CapacitorSQLite query 81963020
as well as:
⚡️  TO JS {"values":[{"ios_columns":[<array of my database's column names here>]

This appears to happen every time I call SQLiteDBConnection.query.  My hope is to have the console completely empty except for specific output around the code I'm currently testing (plus warnings/errors of imported modules, of course).  Am I doing something wrong?  Thanks in advance.

**Comment by jepiqueau** [comment]
@leemcdole  in the capacitor.config.ts under webDir put loggingBehavior: 'none',

**Comment by leemcdole** [comment]
@jepiqueau thanks for the reply!  I'm finding that setting loggingBehavior: 'none' causes ALL console output to stop, even console.log calls I'm making in my code.  To clarify, I want all libraries (specifically sqlite) to stop logging, so I can see my own logs exclusively.

**Comment by jepiqueau** [comment]
@leemcdole this is not related to @capacitor-communnity/sqlite talk to the capacitor team

**Comment by leemcdole** [comment]
@jepiqueau really?  it seems like any library (like @capacitor-community/sqlite) shouldn't be spamming the console log during normal operation.  Errors I would understand.  But this is just regular queries dumping hundreds of characters to the log.

**Comment by jepiqueau** [comment]
@leemcdole show me what you get to the console

**Comment by leemcdole** [comment]
It's the output specified in my OP, just over and over again every time I submit a query.

**Comment by jepiqueau** [comment]
@leemcdole can you send me a copy of your output
